test/js_test.go

```diff
@@ -570,6 +576,13 @@ func TestJetStreamManagement(t *testing.T) {
 		t.Fatalf("ConsumerInfo is not correct %+v", ci)
 	}
 
+	_, err := js.AddConsumer("foo", &nats.ConsumerConfig{
```

Comment:
Make it a one line: `if _, err := js.AddConsumer(...); err != nats.ErrInvalidDurableName {`.
Otherwise, change `:=` to `=` since err is already defined.